cpp/src/io/utilities/hostdevice_vector.hpp

```diff
@@ -173,7 +172,7 @@ class hostdevice_vector {
   }
 
  private:
-  cudf::detail::rmm_host_vector<T> h_data;
+  cudf::detail::host_vector<T> h_data;
```

Comment:
Can you please clarify again (I forget it): why can't we just use a pinned vector for both host and device access, instead of using separate host and device vectors like this?